docs/how-to/container.md

```diff
@@ -0,0 +1,125 @@
+# dcm2bids with Docker and Apptainer / Singularity
+
+We provide a container image that includes both dcm2niix and dcm2bids as well as pydeface and the BIDS validator. You can find it on [Docker Hub](https://hub.docker.com/r/unfmontreal/dcm2bids).
+
+You can install it using [Docker](https://www.docker.com/get-started) or [Apptainer/Singularity](https://www.apptainer.org).
+
+## Prerequisites
+
+Before you begin, make sure you have at least one of the following installed:
+
+- Docker: [Download and install Docker](https://www.docker.com/get-started)
+- Apptainer, formerly known as Singularity : [Download and install Apptainer](https://apptainer.org/docs/admin/main/installation.html)
+
+!!! note
+    If you are using a HPC cluster, Apptainer is the recommended choice and is probably installed on your system. Simply load the module (e.g., 
+    `module load apptainer`) and use the `apptainer` command,
+
+## Step 1: Pull the dcm2bids container
+
+To start, you can either pull the dcm2bids image from the Docker Hub repository or [build it from the Dockerfile in the repository](https://github.com/UNFmontreal/Dcm2Bids/blob/dev/Dockerfile).:
+
+=== "Docker"
+
+    ```
+    docker pull unfmontreal/dcm2bids:latest
+    ```
+
+=== "Apptainer/Singularity"
+
+    ```
+    apptainer pull dcm2bids.sif docker://unfmontreal/dcm2bids:latest
+    ```
+
+## Step 2: Test dcm2bids
+
+The default command, or the point of entry, for the container is `dcm2bids`. So every time you run the container, you can pass the `dcm2bids` arguments and options directly. To test the container, run the following command to display the help message for the `dcm2bids` command.
+
+=== "Docker"
+
+    ```
+    docker run --rm -it unfmontreal/dcm2bids:latest --help
+    ```
+
+=== "Apptainer/Singularity"
+
+    ```
+    apptainer run dcm2bids.sif --help
+    ```
+
+## Step 3: Run `dcm2bids_scaffold`
+
+To run `dcm2bids_scaffold`, `with singularity`, you need to *execute* a command instead of *running* the pre-specified command (`dcm2bids`). You need to bind the respective volumes.
```

Comment:
```suggestion
To run `dcm2bids_scaffold` with Apptainer/Singularity, you need to *execute* a command instead of *running* the pre-specified command (`dcm2bids`). You need to bind the respective volumes.
```